news/9243.bugfix.rst

```diff
@@ -0,0 +1,2 @@
+New resolver: Pick up hash declarations in constraints files and use them to
+filter available distributions.
```

Comment:
```suggestion
Filter available distributions using hash declarations from constraints files.
```

Reply:
I don't think we need to say "new resolver" anymore -- the resolvelib-based resolver has been an opt-out for over 1.5 years at this point.